.github/CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@
 
 This repository contains Terraform core, which includes the command line interface and the main graph engine. 
 
-Providers are implemented as plugins that each have their own repository linked from the [Terraform Registry index](https://registry.terraform.io/browse/providers). Instructions for developing each provider are usually in the associated README file. For more information, see [the provider development overview](https://www.terraform.io/docs/plugins/provider.html).
+Providers are implemented as plugins that each have their own repository linked from the [Terraform Registry index](https://registry.terraform.io/browse/providers). Instructions for developing each provider are usually in the associated README file. For more information, see [the provider development overview](https://developer.hashicorp.com/terraform/plugin).
```

Comment:
Thank you so so much for catching these! Also, we can actually shorten these because our site supports directory navigational redirects (I can't think of the actual term right now but close enough 😂 ) :
```suggestion
Providers are implemented as plugins that each have their own repository linked from the [Terraform Registry index](https://registry.terraform.io/browse/providers). Instructions for developing each provider are usually in the associated README file. For more information, see [the provider development overview](/terraform/plugin).
```

@SarahFrench If it isn't too much trouble, would you be okay with removing the `https://developer.hashicorp.com/` from these?

Reply:
Oh, jeeze I'm so sorry I just noticed this isn't an .mdx file! Ignore me 🤦‍♀️

Reply:
No worries! This PR does touch `website/docs/cli/commands/version.mdx` so I can update that link